### DISCUSSION TITLE
Same Chrome folder content causes toolbar buttons "hidden" in one profile, unhidden in another

### DISCUSSION THREAD

**Discussion by nollinvoyd** [xiaoxiaoflood/firefox-scripts#248] [General Support]
There must be some setting I missed to prevent this.  The chrome folder contents create a different result in two profiles.  

In one of the profiles, navigation toolbar looks as you would expect,

In the other profile, the only thing visible to the right of the address bar is that arrow that reveals the buttons, right above the overflow menu. 

![over](https://user-images.githubusercontent.com/20844322/221308976-0d0038cb-4556-4606-957d-aeaeeb772bac.png)

If you choose "customize", that's the only time those buttons actually appear to the right of the address bar.

![over2](https://user-images.githubusercontent.com/20844322/221309240-af0a99d2-a6b7-462d-9627-7bc5cbcdae41.png)

**Comment by xiaoxiaoflood** [comment]
This is caused by Firefox, not userChromeJS. You can "hide" or "unhide" (actually "pin" or "unpin" by right-clicking each-item.

![image](https://user-images.githubusercontent.com/25161346/221317644-5775e4c3-9acf-45b2-8f59-cb6682bbdd29.png)

-

![image](https://user-images.githubusercontent.com/25161346/221317676-31a9d4cd-99c4-4ac6-a65c-3c882b5a09e4.png)
